src/widgets/option-item/ui/option-item.tsx

```diff
@@ -12,9 +15,13 @@ type OptionItemProps = {
 };
 
 export const OptionItem = ({ title, description, linkLabel, href = '' }: OptionItemProps) => (
-  <article key={title} className="option" data-testid="option">
-    <Subtitle className="option-title">{title}</Subtitle>
-    <Paragraph className="option-description">{description}</Paragraph>
+  <article key={title} className={cx('option-item')} data-testid="option-item">
+    <Subtitle fontSize="medium" color="gray">
```

Comment:
You don't need props here because it's default variant

Reply:
Done